state-synchronizer/src/coordinator.rs

```diff
@@ -394,39 +414,31 @@ impl<T: ExecutorProxyTrait> SyncCoordinator<T> {
     /// StateSynchronizer assumes that it's the only one modifying the storage (consensus is not
     /// trying to commit transactions concurrently).
     fn request_sync(&mut self, request: SyncRequest) -> Result<()> {
+        let local_li_version = self.local_state.highest_local_li.ledger_info().version();
+        let target_version = request.target.ledger_info().version();
+        debug!(
+            LogSchema::event_log(LogEntry::SyncRequest, LogEvent::Received)
+                .target_version(target_version)
+                .local_li_version(local_li_version)
+        );
```

Comment:
I don't think this is really important, but I **think** that it shouldn't execute anything in the `debug!()` if debug is disabled.  Though, this reads so much easier.

Reply:
hm i think this comment might be on the version before my force-push...?